kitchen-sink/react/pages/ContactsList.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import {
+  Page,
+  Navbar,
+  NavbarBackLink,
+  List,
+  ListGroup,
+  ListItem,
+} from 'konsta/react';
+
+export default function ContactsListPage() {
+  const isPreview = document.location.href.includes('examplePreview');
+  return (
+    <Page>
+      <Navbar
+        title="Contacts List"
+        left={!isPreview && <NavbarBackLink onClick={() => history.back()} />}
+      />
+      <List strongIos>
+        <ListGroup>
+          <ListItem title="A" groupTitle contacts="true" />
```

Comment:
If it is a boolean attribute the write it as `<ListItem title="A" groupTitle contacts />` and change all items below